src/domains/gps-common/03_storage_account.tf

```diff
@@ -3,37 +3,44 @@
 ##################
 
 # storage
-module "payments_receipt_sa" {
```

Comment:
Has it been removed because it was not being used? Is this the storage migrated to cosmos table?

Reply:
yes we shall removed it